src/partials/products.html

```diff
@@ -0,0 +1,30 @@
+<section class="products">
+  <a class="products__label" id="our_products">100% natural</a>
+  <h2 class="products__header">Products</h2>
+  <div class="products__container">
+    <div class="products__card">
+      <h2 class="products__name">Ice cream</h2>
+      <p class="products__description">
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut
+        labore et <span class="products_description-hidden"> dolore magna aliqua facilisis. </span>
```

Comment:
Shouldn't be "products__description--hidden" ? If my observation is correct do not forget to edit the styles.

Reply:
It didn't work with the modifier, I had to create a separate class and then it worked (still don't know why).

Reply:
Isn't it because it's "products_description-hidden" instead of "products__description--hidden"?